pkgs/development/compilers/chicken/5/overrides.nix

```
@@ -1,2 +1,51 @@
-{
+{ pkgs, lib, chickenEggs }:
+let
+  addToBuildInputs = pkg: old: {
+    buildInputs = (old.buildInputs or [ ])
+      ++ (if lib.isList pkg then pkg else [ pkg ]);
+  };
+  addToBuildInputsWithPkgConfig = pkg: old:
+    (addToBuildInputs pkg old) // {
+      nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ pkgs.pkg-config ];
+    };
+  addToPropagatedBuildInputsWithPkgConfig = pkg: old: {
+    propagatedBuildInputs = (old.propagatedBuildInputs or [ ])
+      ++ (if lib.isList pkg then pkg else [ pkg ]);
+    nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ pkgs.pkg-config ];
+  };
+in {
+  breadline = addToBuildInputs pkgs.readline;
+  cmark = addToBuildInputs pkgs.cmark;
+  epoxy = addToPropagatedBuildInputsWithPkgConfig pkgs.libepoxy;
+  exif = addToBuildInputs pkgs.libexif;
+  expat = addToBuildInputs pkgs.expat;
+  glfw3 = addToBuildInputsWithPkgConfig pkgs.glfw3;
+  imlib2 = addToBuildInputsWithPkgConfig pkgs.imlib2;
+  magic = addToBuildInputs pkgs.file;
+  mdh = addToBuildInputs pkgs.pcre;
+  nanomsg = addToBuildInputs pkgs.nanomsg;
+  openssl = addToBuildInputs pkgs.openssl;
+  rocksdb = addToBuildInputs pkgs.rocksdb;
+  sdl-base = addToBuildInputs pkgs.SDL;
+  sdl2 = addToBuildInputs pkgs.SDL2;
```

I guess something like this should also work:

```suggestion
  sdl2 = addToPropagatedBuildInputs pkgs.SDL2;
  sdl2-image = addToBuildInputs pkgs.SDL2_image;
  sdl2-ttf = addToBuildInputs pkgs.SDL2_ttf;
```

---

Good idea! I have tried fixing sdl earlier, but couldn't get it to work. That works now.